packages/components/src/components/editor/condition-builder/conditionBuilder.stories.tsx

```diff
@@ -0,0 +1,40 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import { useState } from 'react';
+import {
+  generateConditionString,
+  logicalOperatorOptions,
+  typeOptions
+} from '@/components/editor/condition-builder/condition-builder-utils';
+import { ConditionBuilderProvider } from '@/components/editor/condition-builder/conditionBuilderContext';
+import { ConditionBuilder } from '@/components/editor/condition-builder/conditionBuilder';
+
+const meta: Meta<typeof ConditionBuilder> = {
+  title: 'Editor/ConditionBuilder',
+  component: ConditionBuilder
+};
+
+export default meta;
+
+type Story = StoryObj<typeof ConditionBuilder>;
+
+export const Default: Story = {
+  render: () => {
+    const [condition, setCondition] = useState<string>();
+    return (
+      <>
+        <ConditionBuilderProvider
+          generateConditionString={generateConditionString}
+          logicalOperatorOptions={logicalOperatorOptions}
+          typeOptions={typeOptions}
+        >
+          <ConditionBuilder onChange={setCondition} />
+        </ConditionBuilderProvider>
+        <pre
+          style={{ border: 'var(--basic-border)', maxHeight: '80px', padding: 'var(--input-padding)', borderRadius: 'var(--border-r2)' }}
```

Comment:
maybe use the known variables?
```suggestion
          style={{ border: vars.border.basic, maxHeight: '80px', padding: vars..., borderRadius: vars.border.r2' }}
```